### DISCUSSION TITLE
error caught during app initialization TypeError: Cannot set property $pinia of #<Object> which has only a getter

### DISCUSSION THREAD

**Discussion by Lionberg** [vuejs/pinia#2781] [Help and Questions]
### Reproduction

impossible to add nuxt.config.ts in playground

### Steps to reproduce the bug

Follow instructions at the docs: https://pinia.vuejs.org/ssr/nuxt.html
1. install: 
npx nuxi@latest module add pinia

2. add to package.json:
"overrides": {
  "vue": "latest"
}

3. // nuxt.config.js
export default defineNuxtConfig({
  // ... other options
  modules: [
    // ...
    '@pinia/nuxt',
  ],
})

4. Get the error in console:
error caught during app initialization TypeError: Cannot set property $pinia of #<Object> which has only a getter

P.S. These instructions says nothing about creating plugin and defining a store. Maybe docs needs a link there?

### Expected behavior

 use your store as usual

### Actual behavior

impossible to  use your store as usual -- client error, and error in nuxt:  Cannot set property $pinia of #<Object> which has only a getter

### Additional information

removing  '@pinia/nuxt' from modules solves the issue.
But pinia does not keep state during page refreshes

**Comment by u007** [comment]
yes, i encounter this too, if i dont declare plugin to use it, im getting getActivePinia without app.use(pinia)}

**Comment by u007** [comment]
i think this is due to @pinia/nuxt
